docs/DeveloperGuide.md

```diff
@@ -432,7 +432,16 @@ _{more aspects and alternatives to be added}_
 - Modify the format of `DATE` to be `d-M-yyyy` and `DATETIME` to be `d-M-yyyy`T`H:mm` instead.
 - This allows FINDvisor to accept both single and double-digits day, month and hour values as valid `DATE` and `DATETIME` values and would not require users to pad these single digit values with a leading zero.
 
-### 3. Show warning to user when scheduling an overlapping meeting
+### 5. Specify error message for reschedule and schedule commands
+
+**Current Implementation**:
+- When users input a invalid date for any parameters, the error message states that it is an invalid command format, although the issue is that an invalid date is passed.
```

Comment:
```suggestion
- When users input an invalid `START_DATETIME` or `END_DATETIME`, an invalid command format error message is shown instead of an invalid datetime error message.
- The error message is not representative of the error and should be more specific about which fields are incorrect.
```
Maybe this is a better phrasing?

Reply:
Clearer!